.github/support.md

```diff
@@ -0,0 +1,26 @@
+## Are you having trouble getting started?
+If you haven't already, please checkout our [documentation](https://docs.bugsnag.com/platforms/go/) for guides, references and tutorials.
+
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour 
+
+* Bugsnag version:
+* Go version:
```

Comment:
Might be a good idea to ask for OS as well, since certain platforms lack certain features (e.g. panicwrap).